Packs/ApiModules/Scripts/IAMApiModule/IAMApiModule.py

```diff
@@ -249,6 +258,15 @@ def update_with_app_data(self, app_data, mapper_name, incident_type=None):
             app_data = safe_load_json(app_data)
         self._user_profile = demisto.mapObject(app_data, mapper_name, incident_type)
 
+    def get_first_available_iam_user_attr(self, iam_attrs: List[str], use_old_user_data: bool = True):
```

Comment:
we need to check if id in iam_attrs separately because it's not included in the outgoing mapper. if there, take the value from self._user_profile